src/modules/Unity/Application/mirsurface.cpp

```diff
@@ -211,6 +211,10 @@ MirSurface::MirSurface(NewWindow newWindowInfo,
     connect(m_surfaceObserver.get(), &SurfaceObserver::attributeChanged, this, &MirSurface::onAttributeChanged);
     connect(m_surfaceObserver.get(), &SurfaceObserver::nameChanged, this, &MirSurface::onNameChanged);
     connect(m_surfaceObserver.get(), &SurfaceObserver::cursorChanged, this, &MirSurface::setCursor);
+    connect(m_surfaceObserver.get(), &SurfaceObserver::hiddenChanged, this, [&](bool) {
+        // Mir determines visibility from it's hidden state.
```

Comment:
"its" is possessive, "it's" is "it is"

Reply:
Well, I evaded this by removing it altogether. :D